docs/api/class-reference.md

```diff
@@ -212,3 +212,18 @@ This object represents a generic Creative. It's used as a parent object for more
 
 - `authority: String`
 - `value: String`
+
+## adParameters<a name="adparameters"></a>
+
+- `value: String`
+- `xmlEncoded: String|null`
+
+## survey<a name="survey"></a>
+
+- `value: String`
+- `type: String|null`
+
+## iconClickFallbackImages <a name="iconclickfallbackimages"></a>
+
+- `id: Number`
+- `url: String`
```

Comment:
This is missleading as iconClickFallbackImages is an array and the object correspond to the  `iconClickFallbackImage` (singular)
Also, can the url equal to null or will it be an empty string ?